libs/features/mas/docs/ccd.html

```diff
@@ -4,16 +4,10 @@
     <meta charset="UTF-8">
     <title>CCD Gallery</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="spectrum.css">
```

Comment:
We should probably mention it in `mas` or `mas.js` documentation, that consumer needs to bring their spectrum css

Reply:
there is this already

https://github.com/adobecom/milo/pull/3289/files#diff-d6a960cce00021b9cdcd7700511c82fae250fb5ac37601f38cefe135814f66a7R9

And we have ccd gallery as an example with Spectrum CSS. Do you think we need to provide more details?